contrib/automation_tests/test_cases/bottom_up_tab.py

```diff
@@ -45,3 +48,123 @@ def _execute(self):
             raise RuntimeError('First child of the first item ("ioctl") '
                                'of the bottom-up view doesn\'t start with "drm"')
         logging.info('Verified that first child of the first item starts with "drm"')
+
+
+class VerifyBottomUpContentForLoadedCapture(E2ETestCase):
+
+    def _verify_row_count(self, tree_view_table: Table, expected_rows: int):
+        self.expect_eq(tree_view_table.get_row_count(), expected_rows,
+                       "Bottom-up view has {} rows".format(expected_rows))
+
+    def _verify_first_rows(self, tree_view_table: Table, expectations: list[list[str]],
+                           hidden_columns: set[int]):
+        for i in range(len(expectations)):
+            for j in range(len(expectations[i])):
+                if j in hidden_columns:
+                    continue
+                expectation = expectations[i][j]
+                self.expect_eq(
+                    tree_view_table.get_item_at(i, j).window_text(), expectation,
+                    "Bottom-up view's cell ({}, {}) is '{}'".format(i, j, expectation))
+
+    def _execute(self, selection_tab: bool = False):
```

Comment:
Nit: This would be easier to read if you'd split it into multiple smaller methods.

Reply:
Done. It's now more fragmented so please have a look if this is better.